src/sankoff_handler.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019-2022 bito project contributors.
+// bito is free software under the GPLv3; see LICENSE file for details.
+
+#include "sankoff_handler.hpp"
+
+Eigen::Matrix<double, Eigen::Dynamic, SankoffHandler::state_count> SankoffHandler::GenerateLeavesPartials(const size_t node_count, const size_t pattern_idx) {

```

Comment:
Perhaps "GenerateLeafPartials"? Also ccing @davidrich27 to note that here you're just saying "partials" rather than "partial vectors".